kvmagent/kvmagent/test/test_zbs_storage_plugin.py

```
@@ -0,0 +1,489 @@
+
+    def test_luks_clone_converts_plain_source_to_luks_target(self):
+        plugin = zbs_storage_plugin.ZbsStoragePlugin()
+        commands = []
+        removed = []
+        original_errorout = zbs_storage_plugin.bash.bash_errorout
+        original_rm = zbs_storage_plugin.linux.rm_file_force
+        original_secret_channel = zbs_storage_plugin.volume_secret.luks_secret_channel
+        try:
+            zbs_storage_plugin.bash.bash_errorout = lambda cmd: commands.append(cmd) or ""
+            zbs_storage_plugin.linux.rm_file_force = lambda path: removed.append(path)
+            secret_paths = iter(["/tmp/luks-secret-create", "/tmp/luks-secret-convert"])
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/luks-secret-create"

(S108)